src/test/resources/application.yml

```
@@ -0,0 +1,21 @@
+    database-platform: org.hibernate.dialect.H2Dialect
+    show-sql: true # 테스트 시 SQL 쿼리 확인
+
+  datasource:
+    url: jdbc:h2:mem:testdb # 테스트용 인메모리 DB
+    driver-class-name: org.h2.Driver
+    username: sa
+    password:
+
+jwt:
+  secret:
+    key: VEVTVF9KV1RfU0VDUkVUX0tFWV9URVNUX0pXVF9TRUNSRVRfS0VZX1RFU1RfSldUX1NFQ1JFVF9LRVk= # 테스트용 시크릿 키
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)